services/content/bulk_delete.go

```diff
@@ -0,0 +1,57 @@
+package contentservice
+
+import (
+	"github.com/fastschema/fastschema/db"
+	"github.com/fastschema/fastschema/fs"
+	"github.com/fastschema/fastschema/pkg/errors"
+)
+
+func (cs *ContentService) BulkDelete(c fs.Context, _ any) (any, error) {
```

Comment:
This method should return the affected rows count.
```suggestion
func (cs *ContentService) BulkDelete(c fs.Context, _ any) (int, error) {
```